170820 18:57:18 mysqld_safe Logging to '/its1/GB_BT2/jianzuoyi/biosoft/anaconda/mysql-test/var/mysqld.1/data/gbcluster.err'.
170820 18:57:18 mysqld_safe Starting mysqld daemon with databases from /its1/GB_BT2/jianzuoyi/biosoft/anaconda/mysql-test/var/mysqld.1/data
